### ISSUE TITLE
aria-hidden="true" within labels is not honored upon focus of associated input

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4162]
**Reported by james.curd.razorfish on 2014-05-29 20:43**
Often, asterisks are used to denote that an input is required to a sighted user. Since screen reader users have aria-required to communicate this information to them, it seems preferable to not need to listen to the ASCII method of communicating this.

Placing aria-hidden="true" on the spans wrapping the asterisks should prevent them from being read.

When navigating element by element, NVDA recognizes that aria-hidden is applied to the asterisk and does not read it. However, upon entering an input field, NVDA no longer honors the aria-hidden attribute and reads the asterisk as "star".

```
<label for="first_name">First Name<span aria-hidden="true">*</span></label>
<input id="first_name" type="text" aria-required="true">
```

Expected reading upon entering the field: "First Name edit required"
Current reading upon entering the field: "First Name star edit required"

This issue occurs in NVDA with both IE9 and Firefox in Windows 7.
This works properly in JAWS with IE9 as well as VoiceOver for Safari, Firefox, and Chrome.

**Comment by Adriani90** [comment]
@james.curd.razorfish is this issue still reproducible?
cc: @jcsteh if yes, is there already a Mozilla bug for this?

**Comment by Adriani90** [comment]
@Martinator could you try to reproduce it please?

**Comment by Adriani90** [comment]
Testing with NVDA 2019.1.1, This is reproducible only in IE11.
I am closing this in favor of #5833.
